package.json

```
@@ -1,9 +1,9 @@
 {
     "name": "tbtc-dapp",
-    "version": "0.17.6-pre",
+    "version": "0.17.10-dev",
     "dependencies": {
         "@0x/subproviders": "^6.0.8",
-        "@keep-network/tbtc.js": ">0.19.4-pre <0.19.4-rc",
+        "@keep-network/tbtc.js": ">0.19.5-pre <0.19.5-rc",
```

We no longer release the `-pre`-tagged packages.

```suggestion
        "@keep-network/tbtc.js": ">0.19.5-rc <0.19.5-ropsten",
```

And something that is outside of scope of this PR that I noticed when looking at the dependencies:
The dependencies in the `package.json` in `@keep-network/tbtc.js` don't refer to latest `tbtc` packages. I think we should use the `"@keep-network/tbtc": ">1.1.2-dev <1.1.2-ropsten",` dependency there, same as we use in `coverage-pools`. How do you think, @nkuba?